### DISCUSSION TITLE
sveltekit TypeError: immutable

### DISCUSSION THREAD

**Discussion by cinestesia** [sveltejs/kit#6971] [General]
I'm using sveltekit 1.0.0-next.483 running with npm run dev -- --host
 
connecting to an endpoint with a mobile device i get this error:

typeError: immutable
    at Headers.append ([..]node_modules/undici/lib/fetch/headers.js:227:13)

This error **only occurs on mobile device,** connecting to the local net ip address.

the only way I found to unlock this situation, is by commenting the  line of code n. 227 inside node_modules/undici/lib/fetch/headers.js

I have already asked about this problem ( https://stackoverflow.com/questions/73753704/sveltekit-typeerror-immutable ) , with no luck.

Has anyone else experienced this problem?

**Comment by teemingc** [comment]
I recommend creating a reproduction in a repository and submitting an issue for this.
